mautic/mautic
# Add utm_term tag to the legacy Email Builder

https://backlog.acquia.com/browse/MAUT-3371

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We have to have [utm_term tag](https://en.wikipedia.org/wiki/UTM_parameters) in the legacy Email Builder.
This tag should be present as well as the other utm tags.


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a new email.
2. There is no Campaign Term field in the Email Builder.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Open Email builder.
3. Fill in all UTM tags.
4. Create a test email with some links.
5. Send your test email. Make sure that the links contain all the UTM tags as GET parameters including the `utm_term` tag.
